examples/vue-basic/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "engines": {
-    "node": ">= 6 < 10"
+    "node": ">= 6 <= 10"
```

Comment:
This example still uses the older SDK, so it won't work in Node.js v10 and up

Reply:
oh the FL SDK, makes sense